.github/workflows/client-exports.yaml

```diff
@@ -34,6 +34,18 @@ jobs:
         run: |
           mkdir -v -p ~/.local/share/godot/export_templates
           mv /root/.local/share/godot/templates/${GODOT_VERSION}.stable ~/.local/share/godot/export_templates/${GODOT_VERSION}.stable
+      - name: Replace version string
```

Comment:
Maybe we can generalizate this to do it only once. with .github/actions (we do it in my company)

Reply:
That would be nice.

Reply:
if you wish, create a ticket. I can add info about how to do it, usually are super easy.

Reply:
https://github.com/Codename-Recon/Codename-Recon/issues/38